pexpect/_async.py

```diff
@@ -1,11 +1,12 @@
 import asyncio
 import errno
 import signal
+import os
 
 from pexpect import EOF
 
 @asyncio.coroutine
-def expect_async(expecter, timeout=None):
+def Expecter_expect_loop_async(expecter, timeout=None):
```

Comment:
Unless there is a technical reason to change the names of these functions, please revert these.

Reply:
Hello, Red-M.

Thanks for the quick response. 

See my first point...

I need to add some methods to _async.py.
For choosing names, I need a naming convention...
The current convention looks like "random name".

I have several options:
1) leave PR as is...
2) return original method names, and for new methods use random names like "def _lkjdsfldskjds()"
3) use a separate file instead of _async.py

Isn't naming a sufficient technical reason?

Notice how easy it is to compare the synchronous and asynchronous version 
of a method using the schema I suggested:

```
--- REPLWrapper._run_command    2022-02-06 13:00:58.640969771 -0800
+++ REPLWrapper__run_command_async      2022-02-06 13:01:30.472217769 -0800
@@ -1,17 +1,18 @@
-class REPLWrapper(object):
-    def _run_command(self, command, cmdlines, timeout):
+@asyncio.coroutine
+def REPLWrapper__run_command_async(repl, command, cmdlines, timeout=-1):
         res = []
-        self.child.sendline(cmdlines[0])
+    yield from repl.child.sendline(cmdlines[0], async_=True)
         for line in cmdlines[1:]:
-            self._expect_prompt(timeout=timeout)
-            res.append(self.child.before)
-            self.child.sendline(line)
+        yield from repl._expect_prompt(timeout=timeout, async_=True)
+        res.append(repl.child.before)
+        yield from repl.child.sendline(line, async_=True)

         # Command was fully submitted, now wait for the next prompt
-        if self._expect_prompt(timeout=timeout) == 1:
+    prompt_idx = yield from repl._expect_prompt(timeout=timeout, async_=True)
+    if prompt_idx == 1:
             # We got the continuation prompt - command was incomplete
-            self.child.kill(signal.SIGINT)
-            self._expect_prompt(timeout=1)
-            raise ValueError("Continuation prompt found - input was incomplete:\n"
+        repl.child.kill(signal.SIGINT)
+        yield from repl._expect_prompt(timeout=1, async_=True)
+        raise ValueError("Continuation prompt found - input was incomplete\n"
                              + command)
-        return u''.join(res + [self.child.before])
+    return u''.join(res + [repl.child.before])

```